lib/modules/manager/bazel-module/extract.spec.ts

```diff
@@ -180,9 +245,39 @@ describe('modules/manager/bazel-module/extract', () => {
       );
     });
 
-    it('returns bazel_dep and single_version_override dependencies if a version is specified', async () => {
+    it('returns bazel_dep with no version and local_path_override dependencies', async () => {
       const input = codeBlock`
-        bazel_dep(name = "rules_foo", version = "1.2.3")
+        bazel_dep(name = "rules_foo")
+        local_path_override(
+          module_name = "rules_foo",
+          urls = "/path/to/repo",
+        )
+      `;
+      const result = await extractPackageFile(input, 'MODULE.bazel');
+      if (!result) {
+        throw new Error('Expected a result.');
+      }
+      expect(result.deps).toHaveLength(2);
+      expect(result.deps).toEqual(
+        expect.arrayContaining([
+          {
+            datasource: BazelDatasource.id,
+            depType: 'bazel_dep',
+            depName: 'rules_foo',
+            skipReason: 'local-dependency',
+          },
+          {
+            depType: 'local_path_override',
+            depName: 'rules_foo',
+            skipReason: 'unsupported-datasource',
+          },
+        ]),
+      );
+    });
+
+    it('returns bazel_dep with no version and single_version_override dependencies if a version is specified', async () => {
+      const input = codeBlock`
+        bazel_dep(name = "rules_foo")
```

Comment:
can you please leave the version for a test with a version? if you need a check without version, add a new test